src/front/spv/rosetta.rs

```diff
@@ -0,0 +1,24 @@
+use std::{fs, path::Path};
+
+const TEST_PATH: &str = "test-data";
+
+fn rosetta_test(file_name: &str) {
+    if true {
+        return; //TODO: fix this test
```

Comment:
Does this still need to be fixed in this PR or is this for later?

Reply:
this needs more work, and it's not critical to the PR, so we'll fix it in the follow-ups